kontakt-reservasjonsregister-client/src/main/java/no/difi/meldingsutveksling/serviceregistry/krr/DSFClient.java

```diff
@@ -0,0 +1,50 @@
+package no.difi.meldingsutveksling.serviceregistry.krr;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.*;
+import org.springframework.web.client.RestTemplate;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+
+public class DSFClient {
+
+    private static final Logger log = LoggerFactory.getLogger(DSFClient.class);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "log" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1068)